app/src/main/res/layout/activity_main.xml

```diff
@@ -74,14 +74,30 @@
         app:layout_constraintTop_toBottomOf="@id/arrival_station"
         android:onClick="buttonClick"/>
 
-    <view class="androidx.recyclerview.widget.RecyclerView"
-        android:id="@+id/journey_recycler"
+    <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
These declarations of `xmlns:android` and `xmlns:tools` are redundant as already declared in the parent

Reply:
removed